R/infix_spaces_linter.R

```diff
@@ -79,6 +79,40 @@ infix_overload <- data.frame(
 #' @param allow_multiple_spaces Logical, default `TRUE`. If `FALSE`, usage like `x  =  2` will also be linted;
 #'   excluded by default because such usage can sometimes be used for better code alignment, as is allowed
 #'   by the style guide.
+#'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "x<-1L",
+#'   linters = infix_spaces_linter()
+#' )
+#'
+#' lint(
+#'   text = "1:4 %>%sum()",
+#'   linters = infix_spaces_linter()
+#' )
+#'
+#' # okay
+#' lint(
+#'   text = "x <- 1L",
+#'   linters = infix_spaces_linter()
+#' )
+#'
+#' lint(
+#'   text = "1:4 %>% sum()",
+#'   linters = infix_spaces_linter()
+#' )
+#'
+#' lint(
+#'   text = "x <-      2.4",
+#'   linters = infix_spaces_linter(allow_multiple_spaces = TRUE)
```

Comment:
probably better to give an example where this is done for multi-expression alignment

Reply:
I was tiptoeing around it because the issue is again the code that we show won't exactly be the code that is parsed. But maybe we can make an exception once.

I have changed the example.

Reply:
how does it look like

```
code_lines <- "
ab     <- 1L
abcdef <- 2L
"
cat(code_lines)
lint(text = code_lines, ...)
```